### PR TITLE
#27: Add own builded JAR file to perform CAS Login. Based on https://…

### PR DESCRIPTION
**Pull request by bees4ever**
Add own builded JAR file to perform CAS Login.

### CODE REVIEW

**Comment by bees4ever** [comment]
Yes, I just did not know if I can use Java classes combined with Scala, so I created a JAR. But adding the source code is possible. I will do this soon.

**Comment by andrej-sajenko** [comment]
We can add a new submodule in the gradle project that uses java and add the module as a dependency to all other submodules where you require cas.

**Comment by andrej-sajenko** [comment]
I will add a skeleton for you, then you can add your sources inside.

**Comment by bees4ever** [comment]
Thank you!

**Comment by andrej-sajenko** [comment]
Changes are in master.
You will find a new submodule with the name cas-lib without sources inside.
Add the sources, but first create a folder cas-lib/src/main/java/<your packages and java files>.

File: cas-lib/src/main/java/casclientwrapper/CasAuthenticationException.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package casclientwrapper;
+
+/**
+ * Exception to be thrown if there was an Error with Cas Login.
+ */
```
[the remark this answers is not in the document]
Comment (bees4ever) [reply cas-lib/src/main/java/casclientwrapper/CasAuthenticationException.java]: Ok, I am not the author, I will give an exact source where from I copy&paste it. This is all from https://github.com/thm-mni-ii/tals/tree/master/android/app/src/main/java/com/thm/mni/tals. I may have to change something in this code in the next weeks. But yes, I will document this better.

**Pull request merged**